services/director-v2/tests/unit/test_core_settings.py

```diff
@@ -55,4 +57,8 @@ def test_dynamic_sidecar_settings(image: str) -> None:
         ),
     )
     settings = DynamicSidecarSettings(**required_kwards)
-    assert settings.DYNAMIC_SIDECAR_IMAGE == image
+
+    assert (
+        settings.DYNAMIC_SIDECAR_IMAGE
+        == DynamicSidecarSettings.strip_leading_slashes(image)
```

Comment:
question are you not testing something like:
```function == function``` here?

I mean, whatever changes in the ```strip_leading_slashes``` function will never be caught by such an assert right? you are the testing the function with itself.

Reply:
Think I managed to confuse myself here. You are correct.